app/locales/ru.json

```diff
@@ -1,193 +1,193 @@
 {
   "main": {
     "toolTipHeader": "Stretchly - напоминания о перерывах",
-    "microbreakIn": "Mini Break in $t(utils.seconds, {'count': {{seconds}} })",
-    "breakIn": "Long Break in $t(utils.seconds, {'count': {{seconds}} })",
-    "resumingBreaks": "Работа возобновлена",
+    "microbreakIn": "Мини-перерыв через $t(utils.seconds, {'count': {{seconds}} })",
+    "breakIn": "Длинный перерыв через $t(utils.seconds, {'count': {{seconds}} })",
+    "resumingBreaks": "Продолжаем перерывы",
     "downloadLatestVersion": "Загрузить последнюю версию",
-    "toBreak": "Long Break",
-    "toMicrobreak": "Mini Break",
+    "toBreak": "длинному перерыву",
+    "toMicrobreak": "мини-перерыву",
     "skipToTheNext": "Перейти к ближайшему",
     "resume": "Возобновить",
     "pause": "Пауза",
-    "forHour": "час",
-    "for2Hours": "2 часа",
-    "for5Hours": "5 часов",
+    "forHour": "на час",
+    "for2Hours": "на 2 часа",
+    "for5Hours": "на 5 часов",
     "untilMorning": "до утра",
     "indefinitely": "на неопределённый срок",
     "resetBreaks": "Сбросить перерывы",
     "quitStretchly": "Выход из Stretchly",
-    "restoreDefaults": "Restore defaults",
-    "warning": "This will restore app defaults and all your settings will be lost.",
-    "continue": "Continue",
-    "cancel": "Cancel",
+    "restoreDefaults": "Восстановить настройки по умолчанию",
+    "warning": "Это действие вернет настройки к первоначальным, ваши настройки будут утеряны.",
+    "continue": "Продолжить",
+    "cancel": "Отменить",
     "preferences": "Настройки"
   },
   "break": {
-    "postpone": "Postpone this break",
-    "skip": "Skip this break",
+    "postpone": "Отложить",
```

Comment:
Assuming that the context implies the word "break"

Reply:
This is inside the actual Break Window, the bottom part 

![image](https://user-images.githubusercontent.com/4745859/83413819-c1760c00-a41c-11ea-8aa5-81b82862f363.png)